src/main/java/dev/cdevents/CDEvents.java

```diff
@@ -36,6 +36,10 @@ private CDEvents() {
      */
     public static String cdEventAsJson(CDEvent cdEvent) {
         try {
+            if (!validateCDEvent(cdEvent)) {
```

Comment:
Thanks - I think this is fine, perhaps in future, we might add a flag to make validation optional.
In the golang SDK I did not include validation in the JSON rendering, I made a `validate` method available instead, so that it is possible to render and invalid CDEvent to JSON, and the producer can decide whether to validate or not.

I did include a call to `validate` in the cloudevents binding to enforce sending of valid events thought (similar to what you had before). 

I wonder whether instead of moving the validation from `cdEventAsCloudEvent` to `cdEventAsJson`, it would be better to add a call to `validate` in the test?

Reply:
I'm also ok with keeping it as it is today.

Reply:
Agree, even this keeps the behavior consistent.
Reverting to keep the validation only for CloudEvents binding, removed for JSON render and added test assertion to validate CDEvent against schema.